lib/components/chat_history/twitch/message.dart

```diff
@@ -26,6 +27,31 @@ const colors = [
   Color(0xFF00FF7F),
 ];
 
+class _TwtichClipData {
+  final imgUrl;
```

Comment:
Add type info? Also spell out `image`, it's generally better to avoid abbreviations in code.